packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/actions/shuffleThis.yml

```diff
@@ -12,16 +12,12 @@ initialState:
     - anchor: {line: 0, character: 0}
       active: {line: 0, character: 5}
   marks: {}
+fullTargets: [{type: primitive, mark: {type: cursor}, selectionType: token, position: contents, insideOutsideType: inside, modifier: {type: identity}, isImplicit: false}]
```

Comment:
Huh strange our update fixtures moves this up here. I think it used to put it at the end, but I believe now we totally ignore it in code so it just kind of ends up where it ends up? Maybe we should just remove it here. I think recorder doesn't capture it anymore

Reply:
Ok yeah then we should have it bee removed. Can do that in a separate pr.

Reply:
Yeah we should prob just remove them to reduce noise. Pretty sure we removed all references to it in code